cirq-core/cirq/contrib/quimb/grid_circuits.py

```diff
@@ -89,6 +89,7 @@ def _interaction(
     )
 
 
+@cirq._compat.deprecated_class(deadline='v0.16', fix="Use cirq.merge_k_qubit_unitaries")
 class MergeNQubitGates(cirq.PointOptimizer):
```

Comment:
This class does more than `cirq.merge_k_qubit_unitaries`. It calls off to `cirq.drop_negligible_operations` and `cirq.drop_empty_moments` as well. Will existing tests for this optimizer pass if only `cirq.merge_k_qubit_unitaries` is called?

Reply:
Can you point me to where `MergeNQubitGates` optimizer calls off to `cirq.drop_negligible_operations` and `cirq.drop_empty_moments` ? 

Also, the call sites calling `MergeNQubitGates` have been updated to call `cirq.merge_k_qubit_unitaries` and no tests fail right now. But to answer your specific question, `cirq.merge_k_qubit_unitaries` does not call `cirq.drop_negligible_operations` and `cirq.drop_empty_moments` by default.

Reply:
Whoops, i mistook this for `simplify_expectation_value_circuit` before.